demos/react-jsnation/src/index.tsx

```diff
@@ -1,4 +1,4 @@
-import "@starbeamx/devtool";
+// import "@starbeamx/devtool";
```

Comment:
Commented out to temporarily remove the devtool integration due to the JSX Runtime issue.